packages/tools/webpack-fluid-loader/src/multiResolver.ts

```diff
@@ -33,41 +33,38 @@ function getUrlResolver(options: RouteOptions): IUrlResolver {
     switch (options.mode) {
         case "docker":
             assert(options.tenantId !== undefined, "options.tenantId is undefined");
-            assert(options.bearerSecret !== undefined, "options.bearerSecret is undefined");
             return new InsecureUrlResolver(
                 dockerUrls.hostUrl,
                 dockerUrls.ordererUrl,
                 dockerUrls.storageUrl,
                 options.tenantId,
-                options.bearerSecret);
+                options.bearerSecret ?? "");
 
         case "r11s":
             assert(options.tenantId !== undefined, "options.tenantId is undefined");
-            assert(options.bearerSecret !== undefined, "options.bearerSecret is undefined");
             assert(options.fluidHost !== undefined, "options.fluidHost is undefined");
             if (options.discoveryEndpoint !== undefined) {
                 return new InsecureUrlResolver(
                     "",
                     options.discoveryEndpoint,
                     "https://dummy-historian",
                     options.tenantId,
-                    options.bearerSecret);
+                    options.bearerSecret ?? "");
```

Comment:
Based on my limited knowledge, I am not sure if we should use empty string for `r11s`. Maybe better assert undefined? I think E2E tests should cover the `r11s`

Reply:
Actually, yeah that's fair. Probably better to err on the side of increased validation and loosen restrictions as necessary. Will update.